.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Use this template for reporting bugs encountered while using MLflow.
+labels: 'bug'
+title: [BUG]
+---
+Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md)
```

Comment:
For this bug report template, be good to clarify here that users should ask for help debugging their code on SO rather than issues (in case people don't click through to the issue policy)